### PR TITLE
onnxml linearregressor

### LINKED ISSUE #157: add converters for onnx.ml->pyt for linear models
Now that we have skl->pyt done, we need to have corresponding onnx.ml -> pyt converters

### PR DESCRIPTION
**Pull request by ksaur**
Closes #157

### CODE REVIEW

File: tests/test_onnxml_linear_converter.py [hunk 1]
```diff
@@ -78,15 +85,96 @@ def test_logistic_regression_onnxml_binary(self, rtol=1e-06, atol=1e-06):
...
-        onnx_ml_pred, onnx_pred = self._test_regressor(3)
+        onnx_ml_pred, onnx_pred = self._test_linear(3)
 
         # Check that predicted values match
         np.testing.assert_allclose(onnx_ml_pred[1], onnx_pred[1], rtol=rtol, atol=atol)  # labels
         np.testing.assert_allclose(
             list(map(lambda x: list(x.values()), onnx_ml_pred[0])), onnx_pred[0], rtol=rtol, atol=atol
         )  # probs
 
+    def _test_regressor(self, classes):
```
Comment (interesaaat) [inline tests/test_onnxml_linear_converter.py]: Regression does not have classes. Perhaps call it values?

File: tests/test_onnxml_linear_converter.py [hunk 2]
```diff
@@ -78,15 +85,96 @@ def test_logistic_regression_onnxml_binary(self, rtol=1e-06, atol=1e-06):
...
+        onnx_pred = [[] for i in range(len(output_names))]
+        onnx_pred = session.run(output_names, inputs)
+
+        return onnx_ml_pred, onnx_pred
+
+    @unittest.skipIf(
+        not (onnx_ml_tools_installed() and onnx_runtime_installed()), reason="ONNXML test requires ONNX, ORT and ONNXMLTOOLS"
+    )
+    # test ai.onnx.ml.LinearRegressor with 2 classes
+    def test_linear_regression_onnxml_binary(self, rtol=1e-06, atol=1e-06):
```
Comment (interesaaat) [inline tests/test_onnxml_linear_converter.py]: There is no binary nor multiclass in regression. Just call it ...onnxml1 and 2?

File: hummingbird/ml/operator_converters/onnxml_linear.py [hunk 3]
```diff
@@ -75,4 +75,36 @@ def convert_onnx_linear_model(operator, device=None, extra_config={}):
     )
 
 
+def convert_onnx_linear_regression_model(operator, device, extra_config):
```
Comment (interesaaat) [inline hummingbird/ml/operator_converters/onnxml_linear.py:72]: Actually I don't think you need this. The convert linear model should work.

Comment (ksaur) [reply hummingbird/ml/operator_converters/onnxml_linear.py:72]: mmmm...that will  take a small bit of reworking on parsing; let me try.

**Review by interesaaat — APPROVED** [review]

**Pull request merged**